survey/exporter/tex/question2tex_sankey.py

```diff
@@ -4,7 +4,10 @@
 
 from django.utils.translation import gettext_lazy as _
 from pandas.core.frame import DataFrame
-from pysankey import sankey
+try: 
+    from pysankey import sankey
```

Comment:
Looking at the dependencies it seems that it should work (we use ``["pySankeyBeta~=1.2.2"]``).Maybe the fix here would be:

```python
try:
    from pysankey import sankey
except ModuleNotFoundError:
    warnings.warn(
        "Cannot import sankey, please install the package using the sankey extra."
        " (pip install django-survey-and-report[sankey])"
    )
```

Or am I missing something ? Did you install that way and still encounter the problem ?

Reply:
No, you are absolutely right. I had it wrong in my virtualenv... That patch is not necessary anymore. 
Thanks for your work on this!